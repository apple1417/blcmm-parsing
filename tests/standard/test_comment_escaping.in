<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment></BLCMM></comment>
			<comment>&amp;</comment>
			<comment></hotfix></comment>
			<comment>&#00;</comment>
			<comment><hotfix level="None" name="Hotfix"></comment>
			<comment>    <code profiles="default">This is still in the comment</comment>
			<comment><code	>'"'"'"'"'</comment>
			<comment>\/\/\/\/\/\/</comment>
			<comment><!-- This is not an XML comment --></comment>
			<comment><comment	></category><category name="Not really a new category"><comment	></comment>
			<comment></body><head></comment>
			<comment></head></comment>
			<comment>#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!></comment>
		</category>
	</body>
</BLCMM>

#Commands:

