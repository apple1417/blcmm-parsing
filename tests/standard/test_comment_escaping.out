<BLCMM v="1">
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment>&lt;/BLCMM&gt;</comment>
			<comment>&amp;amp;</comment>
			<comment>&lt;/hotfix&gt;</comment>
			<comment>&amp;#00;</comment>
			<comment>&lt;hotfix level="None" name="Hotfix"&gt;</comment>
			<comment>    &lt;code profiles="default"&gt;This is still in the comment</comment>
			<comment>&lt;code	&gt;&apos;&quot;&apos;&quot;&apos;&quot;&apos;&quot;&apos;</comment>
			<comment>\/\/\/\/\/\/</comment>
			<comment>&lt;!-- This is not an XML comment --&gt;</comment>
			<comment>&lt;comment	&gt;&lt;/category&gt;&lt;category name="Not really a new category"&gt;&lt;comment	&gt;</comment>
			<comment>&lt;/body&gt;&lt;head&gt;</comment>
			<comment>&lt;/head&gt;</comment>
			<comment>#&lt;!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!&gt;</comment>
		</category>
	</body>
</BLCMM>
